.github/renovate.json5

```
@@ -51,7 +52,8 @@
         "src/main/resources/versions\\.json",
         "gradle\\.properties",
       ],
-      "groupName": "AGP versions",
+      "commitMessageTopic": "supported AGP versions",
```

I think more accurately we should say:

```suggestion
      "commitMessageTopic": "tested AGP versions",
```

---

I also think that'd be clearer. However, this is more consistent with the JSON's `supportedVersions`. How do you feel about a separate PR changing this, `supportedVersions` and references to it to "tested versions"?

---

I think this would be a good discussion topic for the team sync.

I agree we should consider it for a separate PR though as to not hold this one up.

---

Meantime, I'm going to revert it back to the current phrase, "AGP versions".

```suggestion
      "commitMessageTopic": "AGP versions",
```